### ISSUE TITLE
wpt editor: "zoom to fit" button

### ISSUE THREAD

**Issue by michihdeu** [TravelMapping/Web#320]
I'd love to have a "Zoom to fit" button like it was introduced for HB with #318 

Pressing the button, the zoom level should be changed so that the whole route is displayed

**Comment by yakra** [comment]
I've had the idea for a while, low-priority, of replacing `Load and Pan` with two separate `Pan` and `Zoom` buttons. It'd take two clicks, but a bit more versatile...

**Comment by michihdeu** [comment]
yes, not just changing the zoom level but the route should also be centered. Just like it's done in HB.

Did I get you right, `Pan` would be the same like today's `Load and Pan`? The 2nd click `Zoom` is used to get the same behavior like "zoom to fit". Fine to me.

**Comment by yakra** [comment]
> Did I get you right, `Pan` would be the same like today's `Load and Pan`?

Almost. Same as today, but without the "Load" step.
I think it'd come in handy sometimes to have that separated out, to be able to do the one without the other.

> The 2nd click Zoom is used to get the same behavior like "zoom to fit". Fine to me.

The 2nd click would get that same behavior, yes.

**Comment by michihdeu** [comment]
This would be great. I guess it's less effort.
